developer/src/kmcmpdll/Compiler.cpp

```diff
@@ -1345,13 +1402,12 @@ DWORD ProcessSystemStore(PFILE_KEYBOARD fk, DWORD SystemID, PFILE_STORE sp)
         return CERR_InvalidLanguageLine;
       }
 
-      swprintf(r, szQ - (size_t)(r - q), L"x%04.4x ", n);  // I3481
-
-      p = wcstok_s(NULL, L" ", &context);  // I3481
-      r = wcschr(q, 0);  // I3481
+      u16sprintf(r, szQ - (size_t)(r - q), L"x%04.4x ", n);  // I3481
+      p = u16tok(NULL, sep_s, &context);  // I3481
+      r = (KMX_WCHAR*) u16chr(q, 0);  // I3481
     }
     delete[] sp->dpString;
-    if (*q) *(wcschr(q, 0) - 1) = 0; // delete final space - safe because we control the formatting - ugly? scared?
+    if (*q) *((KMX_WCHAR*) u16chr(q, 0) - 1) = 0; // delete final space - safe because we control the formatting - ugly? scared?
```

Comment:
Would be easier to read in two lines:

```suggestion
    if (*q) {
      *((KMX_WCHAR*) u16chr(q, 0) - 1) = 0; // delete final space - safe because we control the formatting - ugly? scared?
    }
```

Reply:
👍